android/build.gradle

```diff
@@ -54,5 +54,5 @@ android {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    api 'io.sentry:sentry-android:7.11.0'
+    api 'io.sentry:sentry-android:7.12.0-alpha.4'
```

Comment:
Q: do you need to use this alpha release?

Reply:
We will wait for stable release. Should be this Friday.

Reply:
stable versions are out